api/v1/performanceprofile_webhook.go

```diff
@@ -0,0 +1,33 @@
+package v1
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+// log is for logging in this package.
+var performanceprofilelog = logf.Log.WithName("performanceprofile-resource")
+
+const (
+	webhookPort     = 4343
+	webhookCertDir  = "/apiserver.local.config/certificates"
+	webhookCertName = "apiserver.crt"
+	webhookKeyName  = "apiserver.key"
+)
+
+// SetupWebhookWithManager enables Webhooks - needed for version conversion
+func (r *PerformanceProfile) SetupWebhookWithManager(mgr ctrl.Manager) error {
+	bldr := ctrl.NewWebhookManagedBy(mgr).
+		For(r)
+
+	// Specify OLM CA Info
+	srv := mgr.GetWebhookServer()
+	srv.CertDir = webhookCertDir
+	srv.CertName = webhookCertName
+	srv.KeyName = webhookKeyName
+	srv.Port = webhookPort
+
+	return bldr.Complete()
+}
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
anything else we need here? Otherwise let's remove these warnings

Reply:
Will remove as part of sdk-v1 PR

Reply:
Done